### PR TITLE
[Testing:TAGrading] Rubric Grading Interface

### PR DESCRIPTION
**Pull request by manzil-infinity180**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

### What is the current behavior?
Currently no test for rubric/manual grading interface

### What is the new behavior?
Added test for rubric/manual grading interface

### CODE REVIEW

**Comment by manzil-infinity180** [comment]
In future we can verify it throw api test

File: site/cypress/e2e/Cypress-TAGrading/rubric_grading.spec.js
Comment (IDzyre) [inline site/cypress/e2e/Cypress-TAGrading/rubric_grading.spec.js:18]:
```suggestion
        cy.get('[data-testid="component-64"]')
            .should('contain', 'Full Credit')
            .and('contain', 'Minor errors in Read Me')
            .and('contain', 'Major errors in Read Me or Read Me missing');
```

Same with other uses of chained should calls

**Review by IDzyre — CHANGES_REQUESTED** [review]

File: site/public/templates/grading/Component.twig
Comment (manzil-infinity180) [reply site/public/templates/grading/Component.twig:67]: sorry ig not , i will change it

**Comment by IDzyre** [comment]
Once you get the tests to pass, i'll do a final review and then merge if it looks good.

**Review by IDzyre — APPROVED** [review]
Looks good to me, once the new test names have passed, I will merge.

**Pull request merged**
